### PR TITLE
feat(calls): Allow moderators to download a call participants list

### LINKED ISSUE #13453: Moderators must be able to download the participant list as PDF
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

- [x] CSV #13519 
- [ ] PDF
- [x] Frontend

### PR DESCRIPTION
**Pull request by nickvergessen**
### ☑️ Resolves

* Fix #13453 

### 🚧 Tasks

- [x] Download as CSV
- [x] Add docs
- [x] Add tests

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not possible
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: lib/Controller/CallController.php [hunk 1]
```diff
@@ -111,6 +114,53 @@ public function getPeersForCall(): DataResponse {
...
+			return new Response(Http::STATUS_BAD_REQUEST);
+		}
+
+		if ($format !== 'csv' && $format !== 'pdf') {
+			// Unsupported format
+			return new Response(Http::STATUS_BAD_REQUEST);
+		}
+
+		if ($format !== 'csv') {
+			// FIXME Remove once pdf was implemented.
```
Comment (nickvergessen) [inline lib/Controller/CallController.php:144]: *Note:* Currently only CSV is functional

File: lib/Controller/CallController.php [hunk 2]
```diff
@@ -111,6 +114,53 @@ public function getPeersForCall(): DataResponse {
...
+			return new Response(Http::STATUS_BAD_REQUEST);
+		}
+
+		$output = fopen('php://memory', 'w');
+		fputcsv($output, [
+			'name',
+			'type',
+			'identifier',
+		]);
```
Comment (DorraJaouad) [inline lib/Controller/CallController.php:153]: Should we prettify the output? (headers with capital letters) 
image

**Review by DorraJaouad — APPROVED** [review]
Tested 👍🏻

File: lib/Controller/CallController.php (hunk 2 above)
Comment (nickvergessen) [reply lib/Controller/CallController.php:153]: We can do that for the PDF version later. For the CSV I would keep it "ugly"

Comment (Antreesy) [reply lib/Controller/CallController.php:153]: as-is should be less confusing for later use (like import)

**Pull request merged**